src/config/constants.js

```diff
@@ -230,3 +229,8 @@ export const HOST_MAP = {
 };
 
 export const MAX_THUMBNAIL_SIZE = 10 * 1024 * 1024;
+
+export const BUTTON_TYPES = {
+  MENU_ITEM: 'menuItem',
+  ICON_BUTTON: 'iconButton',
```

Comment:
The string here is not the one you put in the default props, maybe use this on as the default prop ?